CWE/events/usa_elections.txt

```diff
@@ -1339,76 +2175,72 @@ country_event = {
 		
 		NOT = { has_country_flag = elections_disabled }
 		
-		ruling_party_ideology = populist
-		
-		#Trump won previously
-		has_country_flag = trump_2016_election
 	}
 
 	mean_time_to_happen = { months = 11 modifier = { factor = 0.1 year = 2017 } }
 
 	option = {
-		name = "Joe Biden wins, as expected"
+		name = "Joe Biden (Moderate Democrat)"
+		
+		random_owned = { #This should give the player or AI the modifier if they don't have a majority support
+			limit = { owner = { AND = { government = democracy tag = USA NOT = { OR = {pop_majority_ideology = progressive pop_majority_ideology = big_tent pop_majority_ideology = liberal } } } } }

```

Comment:
OR = {pop_majority_ideology = progressive pop_majority_ideology = big_tent pop_majority_ideology = liberal }

Missing a space after OR = {

There are multiple other entries with this problem.

Reply:
Fixed.